internal/rest/restgateway.go

```diff
@@ -154,7 +154,14 @@ func (g *Gateway) Start() error {
 	go func() {
 		<-readyToListen
 		log.Printf("HTTP server listening on %s", g.srv.Addr)
-		err := g.srv.ListenAndServe()
+		var err error
+		if tlsConfig != nil {
+			// Under the covers it will use g.srv.TLSConfig
+			// If the cert and key file are not present, it will use the CA from the TLSConfig
+			err = g.srv.ListenAndServeTLS(g.config.HTTP.TLS.ClientCertsFile, g.config.HTTP.TLS.ClientKeyFile)
```

Comment:
The `ClientCertsFile` and `ClientKeyFile` are for the Kafka client component. To collect configurations for the server cert and key files, we should define new config properties. Maybe call them `config.HTTP.TLS.ServerCertsFile` and `config.HTTP.TLS.ServerKeyFile`, with the JSON property names accordingly

Reply:
This configuration is under the HTTP section already , we have a separate configuration for Kafka here https://github.com/hyperledger/firefly-fabconnect/blob/73fc96bfca84c3fd3b16bfdac5bf395bdf783c04/internal/conf/conf.go#L161-L165